tests/integration/test_reddit.py

```diff
@@ -1,7 +1,13 @@
 """Test praw.reddit."""
+
+from praw import Reddit
+from praw.const import USER_AGENT_FORMAT
+from prawcore import Requestor
```

Comment:
To keep things grouped, can you please move the `prawcore` import to after `praw.models.reddit.base`?